13-template.tex

```diff
@@ -47,16 +47,19 @@ \subsubsection{Functional Requirements}
 
 \subsubsection{Non-Functional Requirements}
 \begin{itemize}
-	\item Should be reasonably performant to run on a desktop/laptop
+	\item Should be reasonably performant to run on a desktop/laptop.
+	\item Should meet minimal security requirements for user recognition.
+	\item Should be smart enough to consistenly manage the whole house-system.  
 \end{itemize}
 
 \section{Analysis and domain model}
 
-The application will emulate a domotic controller connected to home devices.
-The home devices will also be emulated, each with its own logic.
-The user will be able to interact with the devices via the controller with a dashboard.
+The application will emulate a domotic environment, some smart devices and facilities monitoring emulation.
```

Comment:
What do you mean with "and facilities monitoring emulation"? Do you mean "and facilities monitoring the emulation"? But I don't think that's the case in our model.

Reply:
Facilities in this context means 'impianti' or 'servizi'

Reply:
Ok I think I understand what you mean ("e servizi/impianti di monitoraggio"), there are two problems:
1. saying emulation again is a repetition
2. the adjective should be before the noun

The final form should be "and monitoring facilities"

Reply:
No, in this case "facilites" is the subject of the phrase "facilities emulation", but yeah, emulation is repeated, therfore should be deleted.